cypress/e2e/search-page-spec.cy.ts

```diff
@@ -1,29 +1,14 @@
 /// <reference types="Cypress" />
 
-const sampleJoke3 = {
-  "current_page": 1,
-  "limit": 20,
-  "next_page": 2,
-  "previous_page": 1,
-  "results": [
-    {
-      "id": "usrcaMuszd",
-      "joke": "What's the worst thing about ancient history class? The teachers tend to Babylon."
-    }
-  ],
-  "search_term": "",
-  "status": 200,
-  "total_jokes": 307,
-  "total_pages": 15
-}
-
-describe('Search View', () => {
+describe('search view', () => {
   beforeEach('', () => {
-    cy.intercept('https://icanhazdadjoke.com/search?term=cat', sampleJoke3)
```

Comment:
During our refactor, let's do some research into whether or not the fetch is happening when the user visits `localhost:3000`. Not sure if that will still fire even though we are directly visiting the `/search` page.